docs/samples/v1beta1/torchserve/README.md

```diff
@@ -0,0 +1,196 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create PV and PVC
```

Comment:
PV/PVC is great but can be hard for a new user just playing around torchserve. We can add another simpler example with gcs storage and I can help upload the model store with required file to the kfserving example bucket, so user can directly create torchserve inference service from there.